CMakeLists.txt

```diff
@@ -14,20 +14,24 @@ pkg_check_modules(DLT REQUIRED automotive-dlt)
 pkg_check_modules(DLT REQUIRED automotive-dlt-c++)
 pkg_check_modules(AUDIOCOMMON REQUIRED AudioCommon )
 
-#add_subdirectory( deps/audio/common )
+SET(GPTP_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
+string(REPLACE " -O3" "" GPTP_CXX_FLAGS "${GPTP_CXX_FLAGS}")
```

Comment:
Couldn't we use -O2? Finally, which is the offending optimisation? We can go with just this (or -O2) for now, but it is nice to have an issue so we can track that on future.
Finally, couldn't you simply do `string(REPLACE " -O3" "" GPTP_CXX_FLAGS "${CMAKE_CXX_FLAGS}")`? Should be clear that `GPTP_CXX_FLAGS` is `CMAKE_CXX_FLAGS` without `-O3`.